rest-service/manager_rest/test/endpoints/test_deployments.py

```diff
@@ -301,10 +304,8 @@ def test_delete_deployment_with_live_nodes_and_ignore_flag(self):
         (blueprint_id, deployment_id, blueprint_response,
          deployment_response) = self.put_deployment(self.DEPLOYMENT_ID)
 
-        delete_deployment_response = self.delete(
-            '/deployments/{0}'.format(deployment_id),
-            query_params={'ignore_live_nodes': 'true'}).json
-        self.assertEquals(deployment_id, delete_deployment_response['id'])
+        response = self._delete_deployment(self.client, deployment_id)
```

Comment:
As I see it, the whole point of this test was to check the ignore_live_nodes param. So you can't just ignore it